articles/active-directory-b2c/troubleshoot-with-application-insights.md

```diff
@@ -72,7 +72,7 @@ To create an instance of Application Insights in your subscription, follow these
    UserJourneyRecorderEndpoint="urn:journeyrecorder:applicationinsights"
    ```
 
-1. If it doesn't already exist, add a `<UserJourneyBehaviors>` child node to the `<RelyingParty>` node. It must be located after `<DefaultUserJourney ReferenceId="UserJourney Id" from your extensions policy, or equivalent (for example:SignUpOrSigninWithAAD" />`.
+1. If it doesn't already exist, add a `<UserJourneyBehaviors>` child node to the `<RelyingParty>` node. It must be located after `<DefaultUserJourney ReferenceId="UserJourney Id" from your extensions policy, or equivalent (for example:SignUpOrSigninWithAAD" />`. If optional child node `<Endpoints>` is present, it must be before `<UserJourneyBehaviors>`. See [RelyingParty schema reference](./relyingparty.md) for the complete order of elements inside it.
```

Comment:
```suggestion
1. If it doesn't already exist, add a `<UserJourneyBehaviors>` child node to the `<RelyingParty>` node. It must be located after `<DefaultUserJourney ReferenceId="UserJourney Id" from your extensions policy, or equivalent (for example:SignUpOrSigninWithAAD" />`. See [RelyingParty schema reference](./relyingparty.md) for a complete order of the **RelyingParty** child elements.
```

Reply:
@iskandersierra  

I've made a small change to this PR. If the change makes sense, please commit it, and then I'll sign off this PR.

Thanks.